client-ui/build.cmd

```diff
@@ -0,0 +1,31 @@
+@echo off
+
+::  Copyright 2019 ABSA Group Limited
+::
+::  Licensed under the Apache License, Version 2.0 (the "License");
+::  you may not use this file except in compliance with the License.
+::  You may obtain a copy of the License at
+::
+::      http://www.apache.org/licenses/LICENSE-2.0
+::
+::  Unless required by applicable law or agreed to in writing, software
+::  distributed under the License is distributed on an "AS IS" BASIS,
+::  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+::  See the License for the specific language governing permissions and
+::  limitations under the License.
+
+
+set NODE_OPTIONS="--max_old_space_size=8192"
+
+echo "Building Spline UI v%SPLINE_VERSION%"
+
+:: Install required Node version
+:: . ./install-node.cmd
```

Comment:
Why is this just a comment?

Reply:
Because there is no `install-node.cmd` script for Windows, and this step is optional as long as a valid NodeJS is already installed on the host. But you're right I'd rather make a proper TODO comment.